apps/SimpleStreamer/main.cpp

```diff
@@ -1,34 +1,72 @@
+/*   1. Redistributions of source code must retain the above         */
+/*      copyright notice, this list of conditions and the following  */
+/*      disclaimer.                                                  */
+/*                                                                   */
+/*   2. Redistributions in binary form must reproduce the above      */
+/*      copyright notice, this list of conditions and the following  */
+/*      disclaimer in the documentation and/or other materials       */
+/*      provided with the distribution.                              */
+/*                                                                   */
+/*    THIS  SOFTWARE IS PROVIDED  BY THE  UNIVERSITY OF  TEXAS AT    */
+/*    AUSTIN  ``AS IS''  AND ANY  EXPRESS OR  IMPLIED WARRANTIES,    */
+/*    INCLUDING, BUT  NOT LIMITED  TO, THE IMPLIED  WARRANTIES OF    */
+/*    MERCHANTABILITY  AND FITNESS FOR  A PARTICULAR  PURPOSE ARE    */
+/*    DISCLAIMED.  IN  NO EVENT SHALL THE UNIVERSITY  OF TEXAS AT    */
+/*    AUSTIN OR CONTRIBUTORS BE  LIABLE FOR ANY DIRECT, INDIRECT,    */
+/*    INCIDENTAL,  SPECIAL, EXEMPLARY,  OR  CONSEQUENTIAL DAMAGES    */
+/*    (INCLUDING, BUT  NOT LIMITED TO,  PROCUREMENT OF SUBSTITUTE    */
+/*    GOODS  OR  SERVICES; LOSS  OF  USE,  DATA,  OR PROFITS;  OR    */
+/*    BUSINESS INTERRUPTION) HOWEVER CAUSED  AND ON ANY THEORY OF    */
+/*    LIABILITY, WHETHER  IN CONTRACT, STRICT  LIABILITY, OR TORT    */
+/*    (INCLUDING NEGLIGENCE OR OTHERWISE)  ARISING IN ANY WAY OUT    */
+/*    OF  THE  USE OF  THIS  SOFTWARE,  EVEN  IF ADVISED  OF  THE    */
+/*    POSSIBILITY OF SUCH DAMAGE.                                    */
+/*                                                                   */
+/* The views and conclusions contained in the software and           */
+/* documentation are those of the authors and should not be          */
+/* interpreted as representing official policies, either expressed   */
+/* or implied, of The University of Texas at Austin.                 */
+/*********************************************************************/
+
+#include <deflect/Stream.h>
+
 #include <string>
 #include <iostream>
 
 #ifdef __APPLE__
-    #include <OpenGL/gl.h>
-    #include <GLUT/glut.h>
-
-    // GLUT is deprecated in 10.9
-#   pragma clang diagnostic ignored "-Wdeprecated-declarations"
-#   pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#  include <OpenGL/gl.h>
+#  include <GLUT/glut.h> // GLUT is deprecated in 10.9
+#  pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#  pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #else
-    #include <GL/gl.h>
-    #include <GL/glut.h>
+#  include <GL/gl.h>
+#  include <GL/glut.h>
 #endif
 
-// DisplayCluster streaming
-#include <deflect/Stream.h>
-
 bool deflectInteraction = false;
 bool deflectCompressImage = true;
 unsigned int deflectCompressionQuality = 75;
-char * deflectHostname = NULL;
+char* deflectHostname = NULL;
```

Comment:
globals indeed, but this is in a simple 1-file GLUT example app, I don't think that there is a need to rewrite that part...